src/datasets/demo_only.py

```diff
@@ -71,14 +98,46 @@ def create_demo_data(env: BaseEnv, train_tasks: List[Task],
         # To prevent cheating by option learning approaches, remove all oracle
         # options from the trajectory actions, unless the options are known
         # (via CFG.included_options or CFG.option_learner = 'no_learning').
-        for act in traj.actions:
-            if act.get_option().parent not in known_options:
-                assert CFG.option_learner != "no_learning"
-                act.unset_option()
+        if CFG.demonstrator == "oracle":
+            for act in traj.actions:
+                if act.get_option().parent not in known_options:
+                    assert CFG.option_learner != "no_learning"
+                    act.unset_option()
         trajectories.append(traj)
         if CFG.make_demo_videos:
             assert monitor is not None
             video = monitor.get_video()
             outfile = f"{CFG.env}__{CFG.seed}__demo__task{idx}.mp4"
             utils.save_video(outfile, video)
     return Dataset(trajectories)
+
+
+def _human_demonstrator_policy(env: BaseEnv, idx: int, num_tasks: int,
+                               task: Task, event_to_action: Callable[
+                                   [State, matplotlib.backend_bases.Event],
+                                   Action],
+                               state: State) -> Action:  # pragma: no cover
+    # Change the backend to one that supports a GUI.
+    cur_backend = matplotlib.get_backend()
+    matplotlib.use("Qt5Agg")
+    # Render the state.
+    caption = (f"Task {idx+1} / {num_tasks}\nPlease demonstrate "
+               f"achieving the goal:\n{task.goal}")
+    fig = env.render_plt(caption=caption)
+    container = {}
+
+    def _handler(event: matplotlib.backend_bases.Event) -> None:
+        container["action"] = event_to_action(state, event)
+
+    keyboard_cid = fig.canvas.mpl_connect("key_press_event", _handler)
+    mouse_cid = fig.canvas.mpl_connect("button_press_event", _handler)
+    # Hang until either a mouse press or a keyboard press.
+    plt.waitforbuttonpress()
+    fig.canvas.mpl_disconnect(keyboard_cid)
+    fig.canvas.mpl_disconnect(mouse_cid)
+    plt.close()
```

Comment:
you call close at each step of the policy? why is that?

Reply:
each call to `render_state_plt` creates a new plt, so i'm just closing it. that's what the docstring of `render_state_plt` tells me to do:
NOTE: Users of this method must remember to call `plt.close()`,
        because this method returns an active figure object!